core/FileIntegrity.php

```diff
@@ -255,13 +255,12 @@ protected static function getDirectoriesFoundInManifest()
             $directory = $file;
 
             // add this directory and each parent directory
-            while( ($directory = dirname($directory)) && $directory != '.' ) {
+            if( ($directory = dirname($directory)) && $directory != '.' ) {
```

Comment:
The `while` was on purpose as it's supposed to add each parent directory to the array. Could you try to output the $directory variable and exit after like 10 or 15 iterations to see what's happening?